feature/bgp/policybase/ate_tests/route_installation_test/route_installation_test.go

```diff
@@ -150,15 +150,23 @@ func bgpCreateNbr(localAs, peerAs uint32, policy string) *telemetry.NetworkInsta
 
 	d := &telemetry.Device{}
 	ni1 := d.GetOrCreateNetworkInstance(*deviations.DefaultNetworkInstance)
-	bgp := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").GetOrCreateBgp()
+	ni_proto := ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := ni_proto.GetOrCreateBgp()
+	ni_proto.Name = ygot.String("BGP")
+	ni_proto.Identifier = telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP
+	ni_proto.Enabled = ygot.Bool(true)
```

Comment:
Enabling of network instance is not required by other vendors. Infact its breaking already running code. Can you add a deviation if this is specifically needed for a particular vendor.

Reply:
Related discussion: https://github.com/openconfig/featureprofiles/pull/745#issuecomment-1314620480